kml/modules/ground_locations_to_kml.py

```diff
@@ -1,22 +1,24 @@
 """
-Module to convert waypoints list to kml document.
+Module to convert ground locations list to kml document.
 """
 import pathlib
 import time
 
 import simplekml
 
+from kml.modules import location_ground
 
-def waypoints_to_kml(waypoints: "list[tuple[float, float]]",
+
+def ground_locations_to_kml(ground_locations: "list[location_ground.LocationGround]",
                      document_name_prefix: str,
                      save_directory: pathlib.Path) -> "tuple[bool, pathlib.Path | None]":
     """
-    Generates KML file from a list of waypoints.
+    Generates KML file from a list of ground locations.
 
     Parameters
     ----------
-    waypoints: list[tuple[float, float]]
-        Waypoint coordinates in decimal degrees (latitude, longitude).
+    ground_locations: list[location_ground.LocationGround]
+        Ground location coordinates in decimal degrees (latitude, longitude).
```

Comment:
Remove this line since it is redundant (the `LocationGround` class has the descriptions now).